server/app.js

```diff
@@ -10,6 +10,7 @@ const indexRouter = require("./routes/index");
 const pingRouter = require("./routes/ping");
 const usersRouter = require("./routes/api/users");
 const authRouter = require("./routes/api/auth");
+const constestRouter = require("./routes/api/contest");
```

Comment:
Minor spelling mistake here, oops! `constestRouter` -> `contestRouter`

Reply:
Done, Thanks I wouldn't have caught that in this life...haha